apps/rule-manager/test/db/DBTest.scala

```diff
@@ -35,5 +36,6 @@ trait DBTest extends BeforeAndAfter { self: Suite =>
 
   after {
     Evolutions.cleanupEvolutions(playDb)
+    scalikejdbcDb.closeAll
```

Comment:
There were sporadic problems in tests where we'd get 'too many connections' from Postgres, I think because we weren't dropping connections. This seems to have fixed.